### ISSUE TITLE
Adjust case study tile hover style

### ISSUE THREAD

**Issue by makenosound** [icelab/berg#61]
I think the hover styles for the case studies could be a little snappier and a little more interesting. We could have an end state _something_ like:

![screen shot 2016-06-15 at 8 40 22 am](https://cloud.githubusercontent.com/assets/4353/16062159/d126107c-32d4-11e6-95b3-1ce7c7b9f18a.png)

Which involved:

* fading in the overlay colour in full
* sliding the text in the from top/bottom respectively with a short fade
* slide-fading out the non-hover teaser line
* fading out the gradient

And it would all feel a bit snappier. What do you think @andyicelab?

**Comment by andyicelab** [comment]
I like this style idea @makenosound. I wonder if we _do_ need an additional `description` field then: `description` being the teaser line, `title` being the `PROJECT for CLIENT` text.

**Comment by makenosound** [comment]
@andyicelab Yep, we’ll need some more data. We probably want _both_ those lines of text to be configurable.
